meshtastic/interfaces/ble/reconnection.py

```
@@ -154,6 +234,25 @@ def _schedule_reconnect(self, auto_reconnect: bool, shutdown_event: Event) -> bo
             raise
         return True
 
+    # COMPAT_STABLE_SHIM: Public compatibility alias; delegates to _schedule_reconnect.
+    def schedule_reconnect(self, auto_reconnect: bool, shutdown_event: Event) -> bool:
```

Ruff (0.15.5)

[warning] Boolean-typed positional argument in function definition

(FBT001)